tests/tdastro/sources/test_sncosmo_models.py

```diff
@@ -18,6 +19,18 @@ def test_sncomso_models_hsiao() -> None:
     assert np.allclose(fluxes, [4.31210900e-20, 7.46619962e-20, 1.42182787e-19])
 
 
+def test_sncomso_models_hsiao_t0() -> None:
+    """Test that we can create and evalue a 'hsiao' model with a nonzero t0."""
+    model = SncosmoWrapperModel("hsiao", amplitude=1.0e-10, t0=10.0)
```

Comment:
same as above, we should have `t0=55000` here. and `[54990]` in L30 (but the fluxes still won't match because redshift=0.5)